freeipa/src/main/resources/freeipa-salt/salt/freeipa/scripts/freeipa_backup

```diff
@@ -260,6 +268,6 @@ elif [[ "${config[backup_platform]}" = "GCP" ]]; then
 fi
 
 doLog "INFO Backup ${PERMISSIONS_CHECK}completed."
-if [[ -n "$PERMISSIONS_CHECK" ]]; then
+if [[ -z "$PERMISSIONS_CHECK" ]]; then
     doStatus "INFO Backup succeeded."
```

Comment:
logic flipped

Reply:
when I added the permissions check yesterday, I got the logic backward here.